coordinator/src/main/scala/com/socrata/datacoordinator/resources/DatasetSecondaryStatusResource.scala

```diff
@@ -42,12 +48,19 @@ case class DatasetSecondaryStatusResource(storeIdOpt: Option[String],
   private def doUpdateVersionInSecondary(req: HttpRequest, storeId: String): HttpResponse = {
     val defaultSecondaryGroups: Set[String] = serviceConfig.secondary.defaultGroups
     val groupRe = "_(.*)_".r
-    storeId match {
-      case "_DEFAULT_" => defaultSecondaryGroups.foreach(ensureInSecondaryGroup(_, datasetId))
+    val found = storeId match {
+      case "_DEFAULT_" => defaultSecondaryGroups.forall(ensureInSecondaryGroup(_, datasetId))
```

Comment:
This doesn't do quite the same thing; `forall` will stop at the first `false`.  Since `ensureInSecondaryGroup` is side-effecting, this is probably not what you want.

You probably want to make this return `true` if at least one of them is `true` (and you don't want to use `exists` for that because it'll stop at the first `true`)